docs/build/isc/v1.1/docs/tutorials/cross-chain-nft-marketplace-part-2.md

```diff
@@ -0,0 +1,297 @@
+https://github.com/iota-community/ISC-Cross-Chain-NFT-Marketplace/blob/main/contracts/MyONFT721.sol
+```
+
+
+
+## Scripts
+
+We will add scripts that perform the following tasks, in order:
+
+1- Deploy the `MyERC721` contract to the BNB Testnet.
+2- Mint an NFT using the `MyERC721` contract.
+3- Deploy the `MyProxyONFT721` contract to the BNB Testnet.
+4- Deploy the `MyONFT721` contract, the reciever, to the ShimmerEVM Testnet.
+5- Configure the `MyProxyONFT721` contract to send NFTs to the `MyONFT721` contract.
+6- Configure the `MyONFT721` contract to receive NFTs from the `MyProxyONFT721` contract.
+7- Send an NFT from the BNB Testnet to the ShimmerEVM Testnet.
+8- List the received NFT on the NFT marketplace.
+
+Please note that the scripts are named according to the tasks they perform, and the chain they are intended to run on. For example, the script that deploys the `MyERC721` contract to the BNB Testnet is named `deploy_erc721_bnb.js`.
+
+
+### deploy_erc721_bnb.js
+This script will deploy the `MyERC721` contract to the BNB Tetnet, and save the contract's address to a file called `MyERC721_BNB.txt`.
+
+
+```javascript reference
+https://github.com/iota-community/ISC-Cross-Chain-NFT-Marketplace/blob/main/scripts/deploy_erc721_bnb.js
+```
+You can run this script with the following command:
+
+```bash
+npx hardhat run scripts/deploy_er721_bnb.js --network bnbTestnet
+```
+
+### mint_nft.js
+
+After you have deployed the `MyERC721` contract, you are ready to mint an NFT using the following script:
+
+```javascript reference
+https://github.com/iota-community/ISC-Cross-Chain-NFT-Marketplace/blob/main/scripts/mint_nft_bnb.js
+```
+You can run the script by executing the following command:
+
+```bash
+npx hardhat run scripts/mint_nft.js --network bnbTestnet
+```
+### deplpy_proxynftonft_bnb.js
+
+Next, deploy the `MyProxyONFT721` contract to the BNB Testnet and save its address in a file called `MyProxyONFT721_BNB.txt` using the following script:
+
+```javascript reference
+https://github.com/iota-community/ISC-Cross-Chain-NFT-Marketplace/blob/main/scripts/deploy_proxyonft_bnb.js
+```
+You can run the script by executing the following command:
+
+```bash
+npx hardhat run scripts/deploy_proxyonft_bnb.js --network bnbTestnet
+```
+
+### deploy_onft721_shimmer.js
+
+Deploy the `MyONFT721` contract to the Shimmer EVM Testnet and save its address in a file called `MyONFT721_Shimmer.txt` using the following script:
+
+```javascript reference
+https://github.com/iota-community/ISC-Cross-Chain-NFT-Marketplace/blob/main/scripts/deploy_onft_shimmer.js
+```
+You can run the script by executing the following command:
+
+```bash
+npx hardhat run scripts/deploy_onft_shimmer.js --network shimmerevm-testnet
+```
+
+### set_trusted_remote_bnb.js
+
+On the BNB Testnet, make a call to the `MyProxyONFT721` contract to set the `MyONFT721` contract as a trusted remote contract. This will allow the `MyProxyONFT721` contract to send NFTs to the `MyONFT721` contract.
```

Comment:
```suggestion
On the BNB Testnet, call the `MyProxyONFT721` contract to set the `MyONFT721` contract as a trusted remote contract. This will allow the `MyProxyONFT721` contract to send NFTs to the `MyONFT721` contract.
```